accounts/static/accounts/css/login.css

```diff
@@ -215,17 +213,30 @@ body {
 
 /* ADAPTIVE  for mobile devices */
 
-@media only screen and (min-width: 320px) and (max-width: 600px) {
+@media only screen and (width >= 320px) and (width <= 600px) {
```

Comment:
![image](https://github.com/user-attachments/assets/e844a92f-efe0-4dfe-a55a-57ec3de0b165)
![image](https://github.com/user-attachments/assets/eaae3dfb-58bd-4b70-832a-cadcfa041d05)

this is good for the most part too and will just continue to get more and more support! i say we keep, is the new standard.